### PR TITLE
multi-anchor normative requirements in spec

### PR DESCRIPTION
**Pull request by isaacvetter**
https://jira.hl7.org/browse/FHIR-37222

### CODE REVIEW

File: input/pagecontent/2-5-EventNotification.md [hunk 1]
```diff
@@ -146,3 +146,7 @@ During a normal shutdown of an application, it SHALL unsubscribe, and provide a
...
 * should [`syncerror`](3-2-1-syncerror.html) events generated by the Hub be distributed only to subscribers of the event which triggered the [`syncerror`](3-2-1-syncerror.html) event to be generated?  However, this could conflict automatically unsubscribe a non-responsive application after the initial [`syncerror`](3-2-1-syncerror.html) is generated and distributed.
 * should all FHIRcast requests trigger  [`syncerror`](3-2-1-syncerror.html) events to be generated by the Hub for an unresponsive application or only when a context change is requested ([FHIR Resource]-open or [FHIR Resource]-close)?
+
+### Hub Generated `open` Events
+
+If a Hub granted subscriptions to different sets of `hub.events` to different applications for the same session, the Hub is responsible for generation of implied open events. When distributing a received event, a hub SHALL ensure open events for the referenced resource types of the received event, are also sent to subscribers. Hubs SHOULD NOT generate and send duplicative events.
```
Comment (GinoCanessa) [inline input/pagecontent/2-5-EventNotification.md:152]: First sentence:
* Should this be "If a Hub *grants*..."?
Second sentence:
* Should the "open" in "ensure open events" be formatted as `open`?
* I do not think the last comma is necessary (e.g., "...received event are also sent to subscribers."

File: input/pagecontent/4-5-multi-anchor-considerations.md [hunk 2]
```diff
@@ -16,7 +16,11 @@ For example, an [`encounter-open`](3-4-1-encounter-open.html) implies a [`patien
 
 ### Hub derives open events
 
-The hub is responsible for identifying and sending these implied open events. When distributing a received event, a hub SHALL ensure open events are also sent to subscribers, for the referenced resource types of the received event. Hubs SHOULD NOT generate and send duplicative events. 
+The hub is responsible for identifying and sending these implied open events. When distributing a received event, the Hub is responsible for generating and communicating open events for the resource types referenced by the received event. It's important that Hubs don't generate and send duplicative events.
```
Comment (GinoCanessa) [inline input/pagecontent/4-5-multi-anchor-considerations.md:19]: * Should the "open" in "communicating open events" be formatted as `open`?
* Last sentence, the spec generally avoids contractions - *could* expand these (fine either way, just calling attention).

File: input/pagecontent/4-5-multi-anchor-considerations.md [hunk 3]
```diff
@@ -16,7 +16,11 @@ For example, an [`encounter-open`](3-4-1-encounter-open.html) implies a [`patien
 
 ### Hub derives open events
 
-The hub is responsible for identifying and sending these implied open events. When distributing a received event, a hub SHALL ensure open events are also sent to subscribers, for the referenced resource types of the received event. Hubs SHOULD NOT generate and send duplicative events. 
+The hub is responsible for identifying and sending these implied open events. When distributing a received event, the Hub is responsible for generating and communicating open events for the resource types referenced by the received event. It's important that Hubs don't generate and send duplicative events. 
+
+See details in the specification about:
+* Hubs can [reject subscriptions](2-4-Subscribing.html#subscription-response) according to their own internal business logic.
+* Hubs are [required](2-5-EventNotification.html#hub-generated-open-events) to derive and send open events.
```
Comment (GinoCanessa) [inline input/pagecontent/4-5-multi-anchor-considerations.md:23]: Should the "open" in "send open events" be formatted as `open`?

File: input/pagecontent/2-5-EventNotification.md (hunk 1 above)
Comment (isaacvetter) [inline input/pagecontent/2-5-EventNotification.md:152]:
```suggestion
If a Hub grants subscriptions to different sets of `hub.events` to different applications for the same session, the Hub is responsible for generation of implied open events. When distributing a received event, a hub SHALL ensure open events for the referenced resource types of the received event are also sent to subscribers. Hubs SHOULD NOT generate and send duplicative events.

```
```
1. [x] Should this be "If a Hub grants..."?
2. [] Should the "open" in "ensure open events" be formatted as open?
3. [x] I do not think the last comma is necessary (e.g., "...received event are also sent to subscribers."
```

I honestly don't know about #2. We're not referencing a specific event, which is usually when we (somewhat inconsistently) format.

File: input/pagecontent/4-5-multi-anchor-considerations.md (hunk 3 above)
Comment (isaacvetter) [reply input/pagecontent/4-5-multi-anchor-considerations.md:23]: I'm gonna say no. See: https://build.fhir.org/ig/HL7/fhircast-docs/2-3-Events.html

We're only formatting specific events, or other terms that are placed in messages or code.  Any concerns, Gino?

File: input/pagecontent/4-5-multi-anchor-considerations.md (hunk 2 above)
Comment (isaacvetter) [inline input/pagecontent/4-5-multi-anchor-considerations.md:19]:
```suggestion
The hub is responsible for identifying and sending these implied open events. When distributing a received event, the Hub is responsible for generating and communicating open events for the resource types referenced by the received event. It is important that Hubs do not generate and send duplicative events. 

```

**Comment by isaacvetter** [comment]
Thank you for the detailed review, @GinoCanessa ! Wiling to take another look and approve if appropriat?

**Review by GinoCanessa — APPROVED** [review]
Looks great, thanks so much!

**Pull request merged**